.circleci/config.yml

```diff
@@ -1123,24 +1178,13 @@ workflows:
           requires: [ "Framework tests" ]
           framework_target: wordpress_no_ddtrace
           name: "WordPress baseline testsuite"
-      - verify_package:
-          requires: [ "package extension" ]
-          name: "webdevops/php:7.1"
-          docker_image: webdevops/php:7.1
-          package_type: deb
-      - verify_package:
-          requires: [ "package extension" ]
-          name: "php:7.3"
-          docker_image: php:7.3
-          package_type: deb
       - verify_alpine:
           requires: [ "package extension" ]
           matrix:
             parameters:
               php_package:
-                - php
+                - php php-fpm
               docker_image:
-                - alpine:3.7
```

Comment:
`php-fpm` is not aliased in 3.7, so we moved this verification (plus `php` which is just an alias for `php5` into the `php5 php5-fpm` matrix just below.